include/sway/server.h

```diff
@@ -86,7 +88,8 @@ struct sway_server {
 	struct wlr_foreign_toplevel_manager_v1 *foreign_toplevel_manager;
 
 	size_t txn_timeout_ms;
-	list_t *transactions;
+	struct sway_transaction *current_transaction;
+	struct sway_transaction *pending_transaction;
```

Comment:
Maybe add comments to explain what these are

Reply:
Added comments for the 4 transaction-related fields.